.github/workflows/test-add-compiler.yaml

```diff
@@ -15,7 +15,7 @@ on:
       version:
         description: 'PE version to install'
         required: true
-        default: '2021.7.3'
+        default: '2021.7.4'
```

Comment:
I think this change makes sense, but it isn't reflected in the PR title nor the commit title.